### PR TITLE
Add some tests and refactor

### PR DESCRIPTION
**Pull request by tejasbubane**
Using `minitest` for testing and some small refactorings.

### CODE REVIEW

File: lib/rbase/columns.rb [hunk 1]
```diff
@@ -1,136 +1,120 @@
...
-      # Column offset from the beginning of the record
-      attr_reader :offset
-      
-      # Column size in characters
-      attr_reader :size
-      
-      # Number of decimal places
-      attr_reader :decimal
+
+      attr_reader :name, :offset, :size, :decimal
```
Comment (maheshm) [inline lib/rbase/columns.rb:31]: You might want to retain the comments

File: lib/rbase/columns.rb [hunk 2]
```diff
@@ -1,136 +1,120 @@
...
           @pack_converter = Encoder.new('utf-8', options[:encoding])
         end
       end
-      
+
       def pack(value)
-	      value = value.to_s
-        value = @pack_converter.en(value) if @pack_converter
-        [value].pack("A#{size}")
+        @encoding ? @pack_converter.en(value) : [value.to_s].pack("A#{size}")
```
Comment (maheshm) [inline lib/rbase/columns.rb]: As per unoptimised code, value in `@pack_converter.en(value)` should also be `to_s`ed before passed

**Comment by tejasbubane** [comment]
@maheshm Fixed

**Comment by maheshm** [comment]
👍

**Pull request merged**